spec/spec_helper.rb

```diff
@@ -28,7 +28,10 @@ def global_stubs_include_recipe
 end
 
 RSpec.configure do |config|
-  config.platform = 'redhat'
-  config.version = '7.1'
   config.include StackstormCookbook::SpecHelper
+
+  config.log_level = :fatal
+  config.formatter = :documentation
+  config.platform = 'ubuntu'
+  config.version = '14.04'
```

Comment:
FYI changing config to `ubuntu 14.04` by default, since it's preferred (first) StackStorm OS.